CONTRIBUTING.md

```diff
@@ -30,4 +30,4 @@ This [blog post by Matthew Rocklin](https://matthewrocklin.com/blog/work/2018/02
 
 [nebari-docs-issues]: https://github.com/nebari-dev/nebari-docs/issues
 [nebari-issues]: https://github.com/nebari-dev/nebari/issues
-[nebari-community]: https://nebari.dev/docs/community
```

Comment:
This page syncs from `nebari-dev/.github`, opened a corrresponding PR here: https://github.com/nebari-dev/.github/pull/33